### PR TITLE
[FEAT] Ledger Integration

### PR DESCRIPTION
**Pull request by gantunesr**
**Development & PR Process**
1. Follow MetaMask [Mobile Coding Standards](https://docs.google.com/document/d/1VJLwTRsUw_5EDq_o8d6sSbXUAYENLurkRitYO45eY5o/edit?usp=sharing)
2. Add `release-xx` label to identify the PR slated for a upcoming release (will be used in release discussion)
3. Add `needs-dev-review` label when work is completed
4. Add `needs-qa` label when dev review is completed
5. Add `QA Passed` label when QA has signed off

**Description**

_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions,_
_1. What is the reason for the change?_
_2. What is the improvement/solution?_

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses https://github.com/MetaMask/metamask-mobile/issues/1237
Progresses https://github.com/MetaMask/mobile-planning/issues/637

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

**Comment by montelaidev** [comment]
I have read the CLA Document and I hereby sign the CLA

File: package.json [hunk 1]
```diff
@@ -145,9 +148,10 @@
     "@metamask/contract-metadata": "^2.1.0",
     "@metamask/controller-utils": "^1.0.0",
     "@metamask/design-tokens": "^1.9.0",
+    "@metamask/eth-sig-util": "^5.0.2",
```
Comment (gantunesr) [inline package.json:151]: Revert before merge

**Comment by edsonayllon** [comment]
I've been looking forward to this!

No reviews?

**Comment by gantunesr** [comment]
Hey @edsonayllon! We're working fixing an issue that's currently a blocker for the Ledger integration on MetaMask mobile, sadly this was a dependency problem with our current version of RN that we hope to fix asap and carry on with the development and testing of the feature

**Comment by bertho-zero** [comment]
Hey @gantunesr! What is blocking? can we have more details about this?

**Comment by gantunesr** [comment]
Hi @bertho-zero, the blocker is related to our current RN version which the team is working into solving right now. The blocker should be resolved once the app is updated to the latest version of RN. Thank you for the patience!

**Pull request closed without merging**